pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -114,16 +134,22 @@ func (h *Helper) TestIP(ip string) error {
 	defer func() {
 		errors.LogError(h.dockerHelper.StopAndRemoveContainer(id))
 	}()
+	return testIPDial(ip)
+}
 
-	// Attempt to connect to test container
-	testHost := fmt.Sprintf("%s:8443", ip)
-	glog.V(4).Infof("Attempting to dial %s", testHost)
-	if err = cmdutil.WaitForSuccessfulDial(false, "tcp", testHost, 200*time.Millisecond, 1*time.Second, 10); err != nil {
-		glog.V(2).Infof("Dial error: %v", err)
-		return err
+func (h *Helper) TestForwardedIP(ip string) error {
+	// Start test server on host
+	id, err := h.runHelper.New().Image(h.image).
+		PortForward(8443, 8443).
```

Comment:
it's safe to assume/require 8443 is available at this point because that's the port we're going to need to claim later anyway?